services/QuillLMS/app/helpers/vitally_school_stats_helper.rb

```diff
@@ -0,0 +1,33 @@
+module VitallySchoolStatsHelper
```

Comment:
Helpers are usually designed for for views.  This would make more sense in a app/models/concern